lib/password_manager_2.rb

```diff
@@ -1,2 +1,46 @@
+require 'date'
+
 class PasswordManager2
+    def initialize
+        @all_entries = []
+    end
+
+    def add(new_service, new_password)
+        if @all_entries.any?{|entry| entry["service"] == new_service}
+            return "ERROR: Service names must be unique"
+        elsif @all_entries.any?{|entry| entry["password"] == new_password}
+            return "ERROR: Passwords must be unique"
```

Comment:
**Suggestion:** When returning a value, the return can be removed to look cleaner (personal preference) like instead of `return "ERROR: Service names must be unique"` it can be `"ERROR: Service names must be unique"` and it will return the same string.

Reply:
Noted! Thanks :)